### DISCUSSION TITLE
Limit Height of Preview on Cameras Screen

### DISCUSSION THREAD

**Discussion by necromancyr** [blakeblackshear/frigate#7307] [Ask A Question] [answered]
I've tried using the LIVE: HEIGHT: config option to limit the height of a specific camera (a doorbell with a vertically oriented image) but it doesn't seem to actually do anything.  

Is there another option that should be used so I can limit the height of a camera preview on the cameras page so there's not one, GIANT preview for my doorbell cam and the rest smaller images that are vertically-resized based on screen width?

**Comment by NickM-27** [comment]
The `live -> height` config only affects the jsmpeg player. The MSE and WebRTC players have a max width set of 1200 px (or somewhere around there)

**Comment by NickM-27** [comment]
If you want a smaller preview it would be best to just use the cameras sub stream as the live view

**Comment by necromancyr** [comment]
Wouldn't that result in the cameras stream being lower quality/smaller universally?  I want the higher resolution video, but the Cameras screen is dominated by the vertically oriented doorbell image (which I believe is just a single snapshot and not live) due to the layout resizing for width only.

**Reply by NickM-27** [reply, marked as answer]
Oh sorry I misunderstood what screen you were talking about. No, there is no way to customize the camera dashboard

**Reply by NickM-27** [reply]
Better support for wide angle and portrait cameras is planned for the future

**Reply by necromancyr** [reply]
Thanks, appreciate the info/help!
